src/core_atmosphere/physics/mpas_atmphys_vars.F

```diff
@@ -122,6 +122,16 @@ module mpas_atmphys_vars
 ! * added the local variables swddir,swddni,and swddif which are output to subroutine rrtmg_swrad and now input
 !   to the updated module_sf_noahdrv.F.
 !   Laura D. Fowler (laura@ucar.edu) / 2023-04-21.
+! * removed the variable f_qv which is not used in any of the ./physics_wrf modules.
+!   Laura D. Fowler (laura@ucar.edu) / 2024-02-13.
```

Comment:
Again, why f_qv is not used? It is surprising.

Reply:
I think it's because, in WRF, T, qv, u, and v are not optional variables. That is, every scheme or suite is supposed to assume they exist, so you don't need those flags for checks. That said, I don't know how dry idealized tests work. Maybe those variables are initialized with zeros.

Reply:
This comment was just my curiosity. It is not a show-stopper.